java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -2791,6 +2791,38 @@ public void testToString() {
     }
   }
 
+  @Test
+  public void testUInt1VectorToString() {
+    try (final UInt1Vector uInt1Vector = new UInt1Vector("uInt1Vector", allocator)) {
+      setVector(uInt1Vector, (byte) 0xff);
+      assertEquals("[255]", uInt1Vector.toString());
+    }
+  }
+
+  @Test
+  public void testUInt2VectorToString() {
+    try (final UInt2Vector uInt2Vector = new UInt2Vector("uInt2Vector", allocator)) {
+      setVector(uInt2Vector, (char) 0xffff);
+      assertEquals("[" + new Character((char) 0xffff) + "]", uInt2Vector.toString());
```

Comment:
lets use the exact string expected please.

Reply:
Thanks for your feedback. Revised accordingly.